__tests__/users.ts

```diff
@@ -1973,3 +1975,270 @@ describe('query referralCampaign', () => {
     );
   });
 });
+
+describe('query personalizedDigest', () => {
+  const QUERY = `
+    query PersonalizedDigest {
+      personalizedDigest {
+        preferredDay
+        preferredHour
+        preferredTimezone
+      }
+  }`;
+
+  it('should require authentication', async () => {
+    await testQueryErrorCode(
+      client,
+      { query: QUERY, variables: {} },
+      'UNAUTHENTICATED',
+    );
+  });
+
+  it('should throw not found exception when user is not subscribed', async () => {
```

Comment:
I think it's a bit weird to throw an exception in this case, isn't it? I'd expect a response that hints that the user is not subscribed. But it's very minor

Reply:
Came to my mind, I'll try to see if there is a better option. 👍

Reply:
Frontend implementation is here https://github.com/dailydotdev/apps/pull/2250, since it is a separate entity I think 404 makes sense. What I did on the frontend is to handle that error case in our dedicated hook. Let me know what you think.